caching-service/src/main/java/org/zowe/apiml/caching/service/infinispan/config/InfinispanConfig.java

```diff
@@ -60,6 +60,8 @@ public class InfinispanConfig {
     private String port;
     @Value("${jgroups.bind.address}")
     private String address;
+    @Value("${jgroups.keyExchange.port:2157}")
```

Comment:
there's a different default than in the start.sh

Reply:
My idea was to leave the original default here and set zowe default to something in zowe port range - 7xxx. Do you think it is a bad idea?

Reply:
I think it's fine, it just brings up the obvious question, maybe a comment will do

Reply:
while I was writing the comment I realized it would be better to use the same default, probably there is no benefit